packages/core/src/components/templates/FieldTemplate/WrapIfAdditionalTemplate.tsx

```diff
@@ -43,7 +30,8 @@ export default function WrapIfAdditional<T = any, F = any>(
     uiSchema,
     registry,
   } = props;
-  const { RemoveButton } = registry.templates.ButtonTemplates;
+  const uiOptions = getUiOptions(uiSchema);
+  const { RemoveButton } = getTemplate("ButtonTemplates", registry, uiOptions);
```

Comment:
Consider restoring this since we don't need the extra overhead of the call as we can't override them in `uiOptions`. You could add this comment which is in many of the other places (but this one)
```
  // Button templates are not overridden in the uiSchema
```